randovania/games/dread/gui/dialog/game_export_dialog.py

```diff
@@ -182,7 +212,7 @@ def _on_update_target_platform(self):
         )
         self.output_tab_widget.setTabVisible(
             self.output_tab_widget.indexOf(self.tab_ftp),
-            target_platform == DreadModPlatform.ATMOSPHERE and False
+            target_platform == DreadModPlatform.ATMOSPHERE
```

Comment:
maybe check for mod manager too?

Reply:
mod manager is a tab-specific setting that only exists on the "SD" tab.

Reply:
in that case, let's just make sure that it's clear to the user that FTP requires mod manager. I didn't look at the actual UI so I'm not sure if it does that already

Reply:
![image](https://user-images.githubusercontent.com/884928/161456022-1a6467d2-85b5-47c6-893e-fe390f29260d.png)